book/website/collaboration/inclusive-events/inclusive-events-content-guide.md

```diff
@@ -0,0 +1,98 @@
+(cl-inclusive-events-accessibility-guide)=
+# Accessible Content Guidance for Inclusive Event
+A critical part of running an event is ensuring that the content of your event is both inclusive and accessible to people with lived experience. 
+
+This section outlines guidance, requirements, and considerations for ensuring that both your promotional content, and your sessions and speakers' presentation content is as accessible as possible for a diverse and inclusive audience.
+
+(cl-inclusive-events-content-accessibility)=
+## Content Accessibility Guidance 
+Ensuring that event communications, including printed content, are accessible to all attendees is critical.
+
+Please review the following recommendations and ensure that any material you have designed for your event meets the accessibility requirements of your atendees. 
+
+#### Use a size 14 (or larger) font for both digital and printed content.
+- A size 14 font is the recommended size to make text information legible for a broad audience and support people with lived experience of visual impairments, and other conditions that affect readability.
+
+#### Use a Sans Serif font such as Arial, Tahoma, Verdana, Helvetica or Monserrat.
+- "Sans Serif" refers to fonts that do not have embellishments or lines on the letters.
+- Using a *Serif* font such as Times New Roman can negatively impact readability and make it harder for people with dyslexia to read information. 
+- Using a *Sans Serif* font is especially important for online content such as emails and websites.
+
+#### Use high contrast colours in all content.
+- The best example of high contrast colours is using dark text on a light background.
+- If you are planning on using different colour palettes to create content and make communication more visually appealing, please ensure that your colours meet Accessibility Standards for Low Vision and Colour Blindness.
+- You may want to consider giving speakers and presenters a guide on how to make their posters and presentations accessible if your event hosts a poster session.
```

Comment:
Maybe we can share tools such as: https://color.adobe.com/create/color-contrast-analyzer

Reply:
Oh I see @PatriLoto comment above now as well!